src/assets/json/data.json

```diff
@@ -1,1107 +1,1110 @@
 {
-  "QuizType": [
+  "Quiz": [
     {
-      "quizId": "htmlcss",
+      "quizId": "Html/CSS",
+      "rules": "<li class=\"container\">There will be a Negative Marking.</li><ul><li><b>Correct Answer </b> gives you <b>1 Point.</b></li><li><b>Incorrcet Answer </b> Deduct your <b> -0.25 Point.</b></li></ul><li>You will be given the time for each question.</li><ul><li>You have <b> 30 secound </b> for each questions.</li></ul><li>After Completion of time for each question , The question will be change automatically.</li><li>If you want to go for next question before time ending then click on the <b>Next button</b></li>",
+      "timer": 30,
+      "PositivePoints": 10,
+      "NegativePoints": 1,
+      "numberOfQuestions": "3",
+      "type": "radio",
```

Comment:
what is use for type here ?

Reply:
type is given for future enhancement as we could change the option type from radio button to checbox.